charts/flyte-core/values.yaml

```diff
@@ -198,6 +198,65 @@ datacatalog:
   # -- Sets priorityClassName for datacatalog pod(s).
   priorityClassName: ""
 
+#
+# FLYTE_AGENT SETTINGS
+#
+
+flyteagent:
+  enabled: true
+  # -- Replicas count for flyteagent deployment
+  replicaCount: 1
+  image:
+    # -- Docker image for flyteagent deployment
+    repository: ghcr.io/flyteorg/flyteagent
+    # -- Docker image tag
+    tag: 1.6.2b1
+    # -- Docker image pull policy
+    pullPolicy: IfNotPresent
+  # -- Default resources requests and limits for flyteagent deployment
+  resources:
+    limits:
+      cpu: 500m
+      ephemeral-storage: 100Mi
+      memory: 500Mi
+    requests:
+      cpu: 10m
+      ephemeral-storage: 50Mi
+      memory: 50Mi
+  # -- Default regex string for searching configuration files
+  configPath: /etc/flyteagent/config/*.yaml
+  # -- Service settings for flyteagent
+  service:
+    annotations:
+      projectcontour.io/upstream-protocol.h2c: grpc
+    type: NodePort
```

Comment:
Any reason we want this to be Nodeport and not ClusterIP. Since the flyteAgent would be running primarily within the same cluster as propeller